internal/validation/validation.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 var (
-	nameRegexp = regexp.MustCompile(`^[a-z][a-z0-9-]{0,39}$`)
+	nameRegexp         = regexp.MustCompile(`^[a-z][a-z0-9-]{0,39}$`)
+	yamlFilenameRegexp = regexp.MustCompile(`^[A-Za-z0-9-_\/\.\,]{1,255}\.(yaml|yml)$`)
```

Comment:
this regexp doesn't work with non-English letters(

Reply:
Discussed in Slack. Made the regex much more tolerant.